### PR TITLE
Ben 434

### PR DESCRIPTION
**Pull request by zoliszeredi**

### CODE REVIEW

File: filer/tests/admin.py [hunk 1]
```diff
@@ -2508,6 +2510,12 @@ def test_multi_files_perms_for_restricted_descendants(self):
         self.assertEqual(File.objects.filter(restricted=False).count(), 0)
         self.assertEqual(Folder.objects.filter(restricted=False).count(), 0)
 
+    def test_truncate_filename(self):
```
Comment (comandrei) [inline filer/tests/admin.py]: We should also that if the user supplied an extension, that is honored.

File: filer/tests/admin.py [hunk 2]
```diff
@@ -2508,6 +2510,12 @@ def test_multi_files_perms_for_restricted_descendants(self):
...
+        jpeg_file = StringIO.StringIO('      JFIF') # jpeg signature
+        jpeg_file.name = 'empty{}'.format(12345)
```
Comment (comandrei) [inline filer/tests/admin.py]: Minor, I don't think we need to use .format here. Just 'empty12345' should do

**Review by comandrei — APPROVED** [review]
We should add a test for keeping the extension provided by the user.

File: filer/tests/admin.py (hunk 2 above)
Comment (zoliszeredi) [reply filer/tests/admin.py]: will do

File: filer/tests/admin.py (hunk 1 above)
Comment (zoliszeredi) [reply filer/tests/admin.py]: sure, can do

**Review by mariasumedre-3pg — APPROVED** [review]
nice :)

**Pull request merged**